server/public/model/metadata.go

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package model
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/blang/semver/v4"
+)
+
+type MetadataType string
+
+const (
+	CurrentMetadataVersion int          = 1
+	ServerMetadata         MetadataType = "mattermost-support-package"
```

Comment:
is this being used anywhere?

Reply:
Not yet, but it was in the spec. I think intention to add this to the main support packet in the future right?